packages/api/src/controllers/asset.ts

```diff
@@ -47,7 +47,10 @@ const app = Router();
 
 const META_MAX_SIZE = 1024;
 
-function validateAssetMeta(meta: Record<string, string>) {
+function validateAssetMeta( meta: Record<string, string>, asset?:WithID<Asset> ) {
+  meta = _({ ...asset.meta, ...meta })
```

Comment:
The asset is not always available so this needs to be `?.`
```suggestion
  meta = _({ ...asset?.meta, ...meta })
```